cmd/gvproxy/main.go

```diff
@@ -126,6 +128,13 @@ func run(ctx context.Context, g *errgroup.Group, config *GvproxyConfig) error {
 	}
 	log.Info("waiting for clients...")
 
+	notificationSender := notification.NewNotificationSender(config.NotificationSocket)
```

Comment:
Initializing `notificationSender` here because `NewNotificationSender` always returns a valid object (a no-op sender when socket is empty), removing the need for nil checks at every `Send()` call.